### ISSUE TITLE
customStylesheet not added

### ISSUE THREAD

**Issue by evanbutera** [BrianDGLS/node-hologram#8]
When setting 

`
customStylesheet: '/assets/hologram.css', 
`
Hologram outputs

```
            <style media="screen">
                /assets/hologram.css
            </style>
```

seems unable to handle the `{{customStylesheet}}` opt correctly
 
Possible solution - include default style as file and not as inline style, handling both as link tags

**Comment by BrianDGLS** [comment]
This should be fixed in 0.1.3.

Can i double check what version you are using?

**Comment by BrianDGLS** [comment]
See https://github.com/BrianDGLS/node-hologram/blob/master/src/templates/layout.hbs#L9

**Comment by evanbutera** [comment]
Im running 0.1.3.

I've seen hbs template, seems that customStyle is included as inline style while should be included as link element.

**Comment by BrianDGLS** [comment]
Ok sorry, I thought I had fixed that.

I'll revisit it.

**Comment by BrianDGLS** [comment]
Fixed in release 0.1.4

https://www.npmjs.com/package/node-hologram
